Automattic/wp-calypso
# Woo: Update edit state on product create/update

This adds reducer code to do the following upon successful product
create:

1. Replace product id placeholders for the real id in any product edits.
2. Remove the product from creates/updates.

To Test: `npm test`
Additional optional testing: Create a new product with variations and observe the redux edit state in dev tools afterwards. The product should be gone from creates, and the product reference in the variation edits should have a real numeric id in it.
